packages/firestore/lite/test/integration.test.ts

```diff
@@ -67,6 +71,26 @@ describe('Firestore', () => {
       'Firestore has already been started and its settings can no longer be changed.'
     );
   });
+
+  it('cannot use once terminated', () => {
```

Comment:
I'd like to suggest to add a test to verify calls to `terminate()` after it is terminated are simply no-op. 

It's obvious now, but might get over looked when the code evolves.

Reply:
Good idea. Added.